### ISSUE TITLE
Remove drop-shadow from embed and add border

### ISSUE THREAD

**Issue by jkeck** [sul-dlss/sul-embed#604]
This should be completed after #601 is merged.

Due to the fact that we're moving to an iframe only solution the drop shadow in not visible.  This is caused because the embed takes up 100% of the frame and the drop-shadow is not part of the calculated width of the element.  If we add a margin around the element to make the drop shadow visible it will make the alignment of the embed not match up with where it did (effectively pushing the embed left by 10px for every existing consumer).

@jvine suggested that we remove the drop-shadow altogether and instead add an explicit border which would match the top border of the panels (EmbedThis, Download, etc).

**Comment by tingulfsen** [comment]
This picture illustrates the issue:

![shadowbox](https://cloud.githubusercontent.com/assets/3740294/16887382/93b1550e-4a8e-11e6-9702-16594800a8a7.png)

The drop-shadow in question is clearly visible at the very bottom of the image, below the three bottom icons. If we just remove it, we'll also lose the left and right borders that delimit the embed. Therefore we need to add left and right margins, probably just like the horizontal margin just above the "Download item" text.

**Comment by jvine** [comment]
Unless I'm missing a later development, this is answered above:
> remove the drop-shadow altogether and instead add an explicit border which would match the top border of the panels (EmbedThis, Download, etc).

On .sul-embed-container, remove the box-shadow, and replace it with border: 1px solid #5f574f. 

Will the iframe then clip the 1px border? I dunno. But we don't want to add margin within the iframe, or the embed will be misaligned with other things on the page.
